src/arguments/channel.ts

```diff
@@ -0,0 +1,19 @@
+import { Argument, Possible, KlasaMessage } from 'klasa';
+import { Channel } from '@klasa/core';
+
+export default class CoreArgument extends Argument {
+
+	public async run(argument: string, possible: Possible, message: KlasaMessage): Promise<Channel> {
+		// Regular Channel support
+		const channelID = Argument.regex.channel.exec(argument);
+		const channel = channelID ? await this.client.channels.fetch(channelID[1]).catch(() => null) : null;
```

Comment:
can't we use those new fancy nullish operators here

Reply:
No.

Reply:
ok. /resolved